content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -124,6 +124,10 @@ Submissions need to be in Markdown format to be used by the [Hugo](https://gohug
 for the blog. There are [many resources available](https://gohugo.io/documentation/) on how to use
 this technology stack.
 
+For illustrations, diagrams or charts, the [figure shortcode](https://gohugo.io/content-management/shortcodes/#figure)
+can be used. For other images, we strongly encourage use of alt attributes; if an image doesn't
+need any alt attrribute, maybe it's not needed in the article at all.
```

Comment:
Wording here is a bit brusque for tech docs (I recognize this as partially quoting me, but I use a different voice and tone for public docs)

Reply:
Good point.
I'll consider this with any further updates.